### DISCUSSION TITLE
Custom icon in MRT_EditActionButtons

### DISCUSSION THREAD

**Discussion by javonthanh** [KevinVandy/mantine-react-table#90] [Q&A]
I want change icon in MRT_EditActionButtons but can't, you can support for me how to edit. Thankyou

**Comment by KevinVandy** [comment]
See the custom icon docs

**Comment by javonthanh** [comment]
I want replace button in Modal (CURD table) to button another as text and
icon
[image: image.png]

Vào Th 3, 18 thg 7, 2023 vào lúc 14:23 Kevin Van Cott <
***@***.***> đã viết:
